### PR TITLE
Add new SVG icons

### LINKED ISSUE #3776: Icons for Compounds and Environment panel buttons
Breaking this off from the "Allow closing environment panel" issue: https://github.com/Revolutionary-Games/Thrive/issues/3693

Now that the environment and compounds panels will have separate icons, it will be much easier to create appropriate icons for each. 

For the Compounds Panel, the panel simply represents the food/nutrients that the cell has stored. So we could represent this with an icon for food/nutrients. 

We could use an apple and wheat stalk like so:
image

Or a chicken drumstick icon like so:
image

And such an icon would then be reused for the Nutrient Panel in the macroscopic stages for continuity.

For environment, @camilomcatasus created some good icons in the linked issue above, but I feel like a petri dish is not the best since it evokes a small artificial environment, and though the globe icon is good we may want to use a globe icon somewhere else (so would not want repeat icons). So I was thinking perhaps we could use a weather icon like so (since I don't imagine it being used elsewhere):
image

Any other thoughts?

### LINKED ISSUE #5329: Create custom icon for siderophore shooting action
as right now it doesn't have an icon for it

### PR DESCRIPTION
**Pull request by HexapodPhilosopher**
**Brief Description of What This PR Does**

Adds .svg versions of existing compound and environment icons.
Adds icons for Compound and Environment windows
Adds siderophore icon.
Adds macro Wood icon.

**Related Issues**

close #3776 
close #5329

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [X] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: assets/textures/gui/bevel/Siderophore.svg
Comment (hhyyrylainen) [inline assets/textures/gui/bevel/Siderophore.svg:1]: As this is an action it should be in the actions sub folder.

File: simulation_parameters/microbe_stage/compounds.json [hunk 1]
```diff
@@ -31,47 +17,36 @@
       "g": 0.39,
       "b": 0.0
     },
-    "IconPath": "res://assets/textures/gui/bevel/ammonia.png"
+    "IconPath": "res://assets/textures/gui/bevel/Ammonia.svg"
   },
-  "phosphates": {
-    "Name": "PHOSPHATE",
+  "atp": {
```
Comment (hhyyrylainen) [inline simulation_parameters/microbe_stage/compounds.json:22]: Could you keep the same order in this file that there was previously? That'd make it much easier to see what was actually changed. (even if technically it doesn't impact anything in the game)

Comment (HexapodPhilosopher) [reply simulation_parameters/microbe_stage/compounds.json:22]: I thought it would be worth alphabetizing the compounds, but I see your point.

Comment (hhyyrylainen) [reply simulation_parameters/microbe_stage/compounds.json]: I guess that could theoretically help, but the file is so long that it is basically the only good way to navigate it is to use "search forward" feature in a text editor.

**Comment by hhyyrylainen** [comment]
Oh yeah, it came up on discord that the old files these replace should be deleted. Because otherwise someone will accidentally use one of the old files somewhere, and then we'll be in a situation where a mix of files from old and new are used at the same time and I don't want to get into that mess.

**Review by hhyyrylainen — APPROVED** [review]
At 720p game resolution I'm seeing slight "haze" around a few of the icons like iron and hydrogen sulfide (but not glucose). But I think this is an improvement, and it is very nice to have SVGs of the icons.

**Pull request merged**